src/translate/ast_literal.rs

```diff
@@ -29,6 +31,7 @@ pub fn translate_ast_literal(sql_value: &SqlValue) -> Result<AstLiteral> {
     })
 }
 
+// TODO: Repalce the functions below with From
```

Comment:
May be.. this comment is no longer needed?

Reply:
Ah! posted wrong
I'll fix it right away